cmd/juju/backups/backups.go

```diff
@@ -100,23 +103,95 @@ var getAPI = func(c *CommandBase) (APIClient, int, error) {
 
 // dumpMetadata writes the formatted backup metadata to stdout.
 func (c *CommandBase) dumpMetadata(ctx *cmd.Context, result *params.BackupsMetadataResult) {
-	// TODO: (hml) 2018-04-26
-	// fix how --quiet and --verbose are handled with backup/restore commands
-	// should be ctx.Verbosef() here
-	fmt.Fprintf(ctx.Stdout, "backup ID:       %q\n", result.ID)
-	fmt.Fprintf(ctx.Stdout, "checksum:        %q\n", result.Checksum)
-	fmt.Fprintf(ctx.Stdout, "checksum format: %q\n", result.ChecksumFormat)
-	fmt.Fprintf(ctx.Stdout, "size (B):        %d\n", result.Size)
-	fmt.Fprintf(ctx.Stdout, "stored:          %v\n", result.Stored)
-
-	fmt.Fprintf(ctx.Stdout, "started:         %v\n", result.Started)
-	fmt.Fprintf(ctx.Stdout, "finished:        %v\n", result.Finished)
-	fmt.Fprintf(ctx.Stdout, "notes:           %q\n", result.Notes)
-
-	fmt.Fprintf(ctx.Stdout, "model ID:        %q\n", result.Model)
-	fmt.Fprintf(ctx.Stdout, "machine ID:      %q\n", result.Machine)
-	fmt.Fprintf(ctx.Stdout, "created on host: %q\n", result.Hostname)
-	fmt.Fprintf(ctx.Stdout, "juju version:    %v\n", result.Version)
+	ctx.Verbosef(c.metadata(result))
+}
+
+func (c *CommandBase) isVerbose() bool {
```

Comment:
It's not everytime VisitAll since there is caching but I'll look into Init() since it is cleaner..

I did run into similar issue before when working on other commands and Visit in the Run was the solution that worked in all scenarios.